docs/installation/server-config/index.rst

```diff
@@ -26,6 +26,10 @@ Kafka Streams and Kafka Client
     These configurations control how Kafka Streams executes queries. These can be provided with the optional ``ksql.streams``
     prefix. For example,  ``ksql.streams.auto.offset.reset`` and ``ksql.streams.cache.max.bytes.buffering``.
 
+    .. important:: Streams configurations specified in the ``ksqlserver.properties`` file are not automatically propagated
```

Comment:
We actually do propagate the streams configurations. They can also be specified using the prefix `ksql.streams`. `ksql.core.streams` is no longer the prefix to use.